astroquery/gaia/tests/test_gaiatap.py

```diff
@@ -103,44 +111,38 @@ def test_show_message():
     GaiaClass(tap_plus_conn_handler=connHandler, datalink_handler=tapplus, show_server_messages=True)
 
 
-def test_query_object(column_attrs, mock_querier):
-    with pytest.raises(ValueError) as err:
-        mock_querier.query_object(skycoord)
-    assert "Missing required argument: width" in err.value.args[0]
-
-    width = 12 * u.deg
-
-    with pytest.raises(ValueError) as err:
-        mock_querier.query_object(skycoord, width=width)
-    assert "Missing required argument: height" in err.value.args[0]
-
-    table = mock_querier.query_object(skycoord, width=width, height=10 * u.deg)
-    assert len(table) == 3
-    for colname, attrs in column_attrs.items():
-        assert table[colname].attrs_equal(attrs)
-    # by radius
-    table = mock_querier.query_object(skycoord, radius=1 * u.deg)
+@pytest.mark.parametrize(
+    "kwargs", ({"width": 12 * u.deg, "height": 10 * u.deg}, {"radius": RADIUS})
+)
```

Comment:
line lengths are 120 for astroquery, would be nice to avoid blacks tendency for ugly and unnecessary line breaks like this.